.github/workflows/chatbot-inference-llama-2-7b-chat-hf.yml

```diff
@@ -16,18 +15,21 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - name: Load environment variables
+        run: cat ~/itrex-actions-runner/.env >> $GITHUB_ENV
```

Comment:
Need to find a way to hide the token in logs.

Reply:
I'll try to fix it in coming PRs.